### PR TITLE
Jdoze/localappdeploy

### PR DESCRIPTION
**Pull request by jasondoze**
Add script to crownapp_local.sh to install dependencies on Darwin or Linux to run app locally.
    
  Add nodejs_install back to repo for reference.
  
  Test crownapp_local.sh in multipass using vm_deploy script.
  
  Appears to work on Darwin and Linux.
  
  Signed-off-by: Jason Doze <dozejason@gmail.com>

### CODE REVIEW

File: nodejs_install.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ else
   echo -e "\n==== Installing NodeJS && NPM && Update Browserlist ====\n"
   sudo apt install -y nodejs  
   sudo apt install -y npm
+  sudo npx update-browserslist-db@latest
```
Comment (stephenlauck) [inline nodejs_install.sh:24]: This should be in your package.json.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: vm_deploy.sh [hunk 2]
```diff
@@ -8,13 +8,13 @@ echo "==== Begin crownapp deploy ===="
...
 
 # Install homebreww
-if ( which brew > /dev/null ) 
-then
-  echo -e "\n==== Brew installed ====\n"
-else 
-  echo -e "\n==== Installing brew ====\n"
-  NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-fi
+# if ( which brew > /dev/null )
```
Comment (stephenlauck) [inline vm_deploy.sh:11]: you can leave this in if it doesn't do anything if brew is already installed. If it's not installed, might be good to have this in here to satisfy the dependency for install gsed.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: nodejs_install.sh (hunk 1 above)
Comment (jasondoze) [reply nodejs_install.sh:24]: It is, I brought this file back from my other crown app repo when I lost it yesterday.

File: vm_deploy.sh (hunk 2 above)
Comment (jasondoze) [reply vm_deploy.sh:11]: I was using this temporarily to test the crownapp_local.sh. the vm deploy we renamed yesterday to crownapp_deploy. I brought this file back also from my other repo.

**Pull request closed without merging**